docs/source/Install/installOnLinux.rst

```diff
@@ -79,6 +84,60 @@ Dependencies
 
 #. A Git compatible version control tool like `SourceTree <http://sourcetreeapp.com>`__ should be used to :ref:`pull/clone <pullCloneBSK>` the Basilisk repository.
 
+
+Fedora
+^^^^^^
+
+#. On Fedora, install the following developer packages and support libraries::
+
```

Comment:
The pre-commit test is failing because there are trailing white spaces.  See https://github.com/AVSLab/basilisk/blob/develop/CONTRIBUTING.md on installing pre-commit testing tools that will correct this.  I have a local branch that fixes this, but I don't have push access privilege to your PR branch.